scania-digital-design-system/sdds
# fix: add tegel font variables to typography

**Describe pull-request**  
Adding Tegel font variables to the typography package

**Solving issue**  
Fixes: [AB#2484](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2484)

**How to test**  
1. Branch out this branch on your local
2. Check if under theme/core/typography you find a file _tegel-font-variables.scss file
3. Run `npm run build` command while in typography package
4. Check if same file is available under "dist" folder

**Additional context**  
We would to come back to the typography package and maybe clean it up a bit.
